static/planttracer.js

```diff
@@ -16,8 +16,7 @@ const UNDELETE_BUTTON='UNDELETE';
 const PLAY_LABEL = 'play';
 const PLAY_TRACKED_LABEL = 'play tracked';
 const UPLOAD_TIMEOUT_SECONDS = 20;
-const crypto = require('crypto'); 
-
+const CryptoJS = require('crypto-js');
```

Comment:
I'm confused. I thought we were removing the `require` statement. Why is it still here?

Reply:
I think the main thing to do for now is to remove the offending test and import until we figure out a way forward

Reply:
Well, you could only use the `require` statement if you are running on node. You could only use the require statement after you validate that it is not present. But right now this code is causing crashes and it needs to be removed from the main branch ASAP.

Reply:
The commit I just made removes the offending line and pauses the use of the test that depends on it